web/info/src/hooks.server.ts

```diff
@@ -1,19 +1,37 @@
-import type { Handle } from '@sveltejs/kit';
+import { APP_THEME_COOKIE_NAME, APP_THEMES } from '$lib/constants';
+
+import type { Handle, ResolveOptions } from '@sveltejs/kit';
 
 export const handle = (async ({ event, resolve }) => {
-  const response = await resolve(event, {
-    // As per https://kit.svelte.dev/docs/hooks#server-hooks-handle, we need
-    // to specify headers to be included in serialized responses when a `load` function
-    // loads a resource server-side with a `RouteFetch`
-    filterSerializedResponseHeaders: (name: string, _value: string) => {
-      switch (name) {
-        case 'content-type':
-          return true;
-        default:
-          return false;
-      }
+  const resolveOptions: ResolveOptions = {};
+
+  resolveOptions.filterSerializedResponseHeaders = (name: string, _value: string) => {
+    switch (name) {
+      case 'content-type':
+        return true;
+      default:
+        return false;
     }
-  });
+  };
+
+  const theme = event.cookies.get(APP_THEME_COOKIE_NAME);
+
+  if (theme && APP_THEMES.includes(theme)) {
+    resolveOptions.transformPageChunk = ({ html }) => {
+      const match = html.match(/(<html.*?)(>)/);
```

Comment:
**note (non-blocking)**: we aren’ŧ guaranteed that the `<html>` opening tag is always on the same line in the page chunk because SvelteKit doesn’t minify for us.

Instead of making this more resilient (see the comment below), we’ll just need to be careful to keep the `<html>` opening tag in one line in `web/info/src/app.html` and not format it with Prettier.

Reply:
Good point. I can try to make the regex here a more resilient